src/patternfly/components/ProgressStepper/examples/ProgressStepper.md

```diff
@@ -323,6 +316,44 @@ cssPrefix: pf-c-progress-stepper
   {{/progress-stepper-step}}
 {{/progress-stepper}}
 ```
+### With popovers
```

Comment:
I would probably call this "with help text" since that's all the core example demos.